problemsets-mat223/linearalbegra.tex

```diff
@@ -2289,13 +2314,57 @@ \section*{Subspaces and Bases}
 	For each set, draw it and explain whether or not it is a subspace of $\R^2$.
 	\begin{parts}
 		\item $A=\{\vec x\in\R^2:\vec x=\mat{a\\0}\text{ for some }a\in\Z\}$.
+			\begin{solution}
+				INSERT PICTURE
```

Comment:
`XXX`, please

Reply:
Done. I've been using `XXXX` everywhere else, so that's what I changed these to. I think I might have done that in one other place too, sorry.